src/azure-cli/azure/cli/command_modules/storage/operations/file.py

```diff
@@ -407,9 +414,34 @@ def _make_directory_in_files_share(file_service, file_share, directory_path, exi
 
 
 def _file_share_exists(client, resource_group_name, account_name, share_name):
-    from azure.core.exceptions import HttpResponseError
     try:
         file_share = client.get(resource_group_name, account_name, share_name, expand=None)
         return file_share is not None
     except HttpResponseError:
         return False
+
+
+# pylint: disable=redefined-builtin
+def generate_sas_file(cmd, client, directory_name=None, file_name=None, permission=None, expiry=None, start=None,
+                      id=None, ip=None, protocol=None, **kwargs):
+    t_generate_file_sas = get_sdk(cmd.cli_ctx, ResourceType.DATA_STORAGE_FILESHARE,
+                                  '_shared_access_signature#generate_file_sas')
+    file_path = file_name
+    if directory_name:
+        file_path = directory_name + '/' + file_path
+    file_path = file_path.split('/')
+    return t_generate_file_sas(account_name=client.account_name, share_name=client.share_name, file_path=file_path,
+                               account_key=client.credential.account_key, permission=permission, expiry=expiry,
+                               start=start, policy_id=id, ip=ip, protocol=protocol, **kwargs)
+
+
+def file_exists(client, **kwargs):
+    try:
+        res = client.get_file_properties(**kwargs)
+        return bool(res)
+    except HttpResponseError:
+        return False
+
+
+def file_updates(client, **kwargs):
+    return client.set_http_headers(**kwargs)
```

Comment:
If no custom logic, just directly use `storage_command`

Reply:
This is to fix the default parameter not allowed for update commands lint rule.